src/axom/core/MapCollection.hpp

```diff
@@ -179,8 +179,15 @@ class MapCollection : public ItemCollection<T>
   ///
   bool hasItem(const std::string& name) const
   {
-    typename MapType::const_iterator mit = m_name2idx_map.find(name);
-    return (mit != m_name2idx_map.end() ? true : false);
+    if(m_name2idx_map.empty())
+    {
+      return false;
+    }
+    else
+    {
+      typename MapType::const_iterator mit = m_name2idx_map.find(name);
```

Comment:
Not too important but it would read better as "const auto mit".

Reply:
Thanks -- I updated all of them